.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug
```

Comment:
Should this be "bug" to match the one pre-created by GitHub? I think they are case-sensitive.